Client/Frontend/Strings.swift

```diff
@@ -153,6 +153,7 @@ extension String {
                 public static let DefaultBrowserTitle = MZLocalizedString("DefaultBrowserCard.Title", tableName: "Default Browser", value: "Switch Your Default Browser", comment: "Title for small card shown that allows user to switch their default browser to Firefox.", lastUpdated: .unknown)
                 public static let DefaultBrowserDescription = MZLocalizedString("DefaultBrowserCard.Description", tableName: "Default Browser", value: "Set links from websites, emails, and Messages to open automatically in Firefox.", comment: "Description for small card shown that allows user to switch their default browser to Firefox.", lastUpdated: .unknown)
                 public static let DefaultBrowserButton = MZLocalizedString("DefaultBrowserCard.Button.v2", tableName: "Default Browser", value: "Learn How", comment: "Button string to learn how to set your default browser.", lastUpdated: .unknown)
+                public static let DefaultBrowserCloseAccessibility = MZLocalizedString("DefaultBrowserCloseButtonAccessibility.v102", value: "Close", comment: "Accessibility label for action denoting closing default browser card.", lastUpdated: .v102)
```

Comment:
I think we'll need to update a couple things here. It's now called the Home Tab Banner and not default browser card (although, you'll see some "Default Browser Card" references here and there).

If it's not tied specifically to default browser card, could you please rename to HomeTabBanner?

Reply:
Yes, just to make sure @nishant2718. Are we only changing the variable names, or the localized keys as well? I'm wondering if changing the key breaks the localization.

Reply:
Just variable name. The key should remain the same the same for the exact reason you mentioned: changing it would break localization.

Reply:
Thanks @electricRGB, should be already updated.

Reply:
I recommend also replacing default browser card to home tab banner in the comment as well, but a minor nit. Looks good!